ci_framework/roles/libvirt_manager/tasks/deploy_layout.yml

```diff
@@ -50,6 +50,37 @@
   loop_control:
     label: "{{ item.key }}"
 
+- name: Find all public key in the default user directory
+  ansible.builtin.find:
+    paths: "{{ ansible_user_dir }}/.ssh"
+    file_type: file
+    patterns: '*.pub'
+  register: users_pub_key
+
+- name: Update authorized keys for existing user
+  ansible.builtin.shell: |
```

Comment:
please use the ansible.posix.ssh_authorized_key: https://docs.ansible.com/ansible/latest/collections/ansible/posix/authorized_key_module.html